dxfReader.py

```diff
@@ -307,13 +311,57 @@ def start_section(cargo):
 			section.data.append(data)
 			data = []
 def end_section(cargo):
-	"""Expects (infile, drawing) as cargo, searches for next section."""
+	"""Expects (infile, drawing, acadVersion) as cargo, searches for next section."""
 	#print "Entering end_section state!"
 	infile = cargo[0]
 	drawing = cargo[1]
+	acadVersion = cargo[2]
+	if sys.version_info >= (3, 0) and not acadVersion:
```

Comment:
just FYI, officially now FreeCAD dropped support for python2. So the sys.version_info check could be removed. There are still some of them in the file, though, so no big deal